infra-aws/README.md

```
@@ -0,0 +1,35 @@
+# k8s
+
+# Steps to create S3 bucket
+
+1. `cd infra-aws/bucket-setup`
+2. Add access key and secret key to provider "aws"{} 
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)